src/App.test.js

```diff
@@ -1,8 +1,6 @@
-import { render, screen } from '@testing-library/react';
+import { render } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
+test('renders empty screen', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
 });
```

Comment:
**issue:** this don't test if the screen is empty

Reply:
done in https://github.com/CaaioSB/react-boilerplate/pull/1/commits/fcf7e92fb5c916a713413b41452dc25e1157e3a5